docs/installation/linux.md

```diff
@@ -0,0 +1,83 @@
+# Linux installation
+
+## General information
+
+Xournal++ is a free and open source program and can be easily compiled from source, although for user convenience it is also distributed via precompiled binary. In case you want to compile xournalpp on your own you should follow the guide on the [github page](https://github.com/xournalpp/xournalpp/blob/master/readme/LinuxBuild.md). If you decide to do otherwise, then this guide is dedicated to that process.
+
+## Options for all/most distros
+
+The following options are available on most major Linux distros.
+
+<ul id="linuxDownloadsContainer" class="downloadsContainer">
+<a class="xournalppButton downloadButton" href="{{downloads.linux.appimage}}">AppImage</a>
+<a class="xournalppButton linkButton" href="{{downloads.linux.flatpak}}">FlatPak</a>
+<a class="xournalppButton linkButton" href="{{downloads.linux.snap}}">Snap Package</a>
+<a class="xournalppButton linkButton" href="{{downloads.allVersions}}">Other versions</a>
+<a class="xournalppButton linkButton" href="{{downloads.nightly}}">Nightly (latest unstable, including nightly AppImage)</a>
+<a class="xournalppButton linkButton" href="#options-for-specific-distros-potentially-preferable">Or use distro specific method</a>
+</ul>
+
+## Options for specific distros (potentially preferable)
+
+On _specific Linux distros_ you have the following extra options:
+
+### Ubuntu and derivatives
+
+* Install the latest *unstable* nightly release from the following ppa
+  
+        sudo add-apt-repository ppa:andreasbutti/xournalpp-master
+        sudo apt update
+        sudo apt install xournalpp
+
+    which you can easily upgrade via `sudo apt update && sudo apt upgrade`
+
+* Install the latest *stable* release from the following *unofficial* ppa
+
+        sudo add-apt-repository ppa:apandada1/xournalpp-stable
+        sudo apt update
+        sudo apt install xournalpp
+
+* Or:
+<ul id="ubuntuContainer" class="downloadsContainer">
+<a class="xournalppButton linkButton" href="{{downloads.linux.ubuntuStable}}">Download stable</a>
+<a class="xournalppButton linkButton" href="{{downloads.nightly}}">Download nightly (latest unstable) version</a>
+<a class="xournalppButton linkButton" href="{{downloads.allVersions}}">Other versions</a>
```

Comment:
This is invalid HTML. The `<a>` tags needs to be wrapped in `<li></li>`, since the only valid children of a `ul` are `li`.